services/print/service/user_service.go

```diff
@@ -0,0 +1,29 @@
+package service
+
+import (
+	"encoding/json"
+	"errors"
+	"github.com/HackIllinois/api/services/print/config"
+	"github.com/HackIllinois/api/services/print/models"
+	"net/http"
+)
+
+/*
+	Get basic user info from user serivce
+*/
+var GetUserInfo = func(id string) (*models.UserInfo, error) {
```

Comment:
It's an interesting idea we could use to mock functions for testing. I'd rather use something like `https://github.com/jarcoal/httpmock` instead though so we don't have to assign all our functions to variables.

Reply:
This is a testing pattern I got from golang itself, lmk if we want to change this. Imo I didn't think these tests required any sort of more complicated mocking.

Reply:
My argument for `https://github.com/jarcoal/httpmock` is that it allows us to mock just the http request to other services instead of the entire function. That's not super relevant here since the function is simple. But I think it's important for more complex functions. And I would prefer to have a single way to mock all inter-service calls across the codebase when unit testing.